src/main/kotlin/org/dif/peerdid/PeerDIDResolver.kt

```diff
@@ -5,7 +5,14 @@ package org.dif.peerdid
 import org.dif.model.DIDDoc
 import org.dif.model.PeerDID
 
-/** Resolves [PeerDID] to [DIDDoc]*/
+/** Resolves [DIDDoc] from [PeerDID]
+ * @param [peerDID] PeerDID to resolve
+ * @param [versionId] a specific version of a [DIDDoc].
+ *  If value is default, version of [DIDDoc] will be latest.
+ *  [versionId] is not used for now, as we support only static layer where [DIDDoc] never changes
+ * @throws IllegalArgumentException if [peerDID] parameter does not match [peerDID] spec
```

Comment:
We also throw that error if a valid DID DOC can not be produced from the peer did, right? Let's mention this in the doc.